.github/scripts/split_spec.sh

```diff
@@ -2,7 +2,10 @@
 set -eou pipefail
 
 echo "Running FOAS CLI versions command"
-foascli versions -s openapi-foas.json -o ./openapi/v2/versions.json --env "${target_env:?}"
```

Comment:
I see that the PR has the `.keep` file. We should remove them and add `.keep` in the gitignore